depmapomics/copynumbers.py

```diff
@@ -187,12 +186,12 @@ def updateTracker(
         lowqual,
         lowqual,
         newgs,
-        sheetcreds,
-        sheetname,
         refworkspace,
         bamfilepaths,
         dry_run,
         samplesinset,
+        trackerobj=trackerobj,
+        gumbo=gumbo,
```

Comment:
I'm surprised to see this gumbo parameter. This code isn't going to work without gumbo, will it?